File: app/src/main/java/com/hhh/onepiece/subscribe/SubscribeViewHolder.java
package com.hhh.onepiece.subscribe;

import android.graphics.drawable.ColorDrawable;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.hhh.mvvm.listener.OnImageShowListener;
import com.hhh.mvvm.recycler.RecyclerViewHolder;
import com.hhh.onepiece.R;
import com.hhh.onepiece.model.WorksModel;
import com.hhh.onepiece.view.NetworkImageView;

public class SubscribeViewHolder extends RecyclerViewHolder<WorksModel> {

  protected NetworkImageView mCoverView;

  public SubscribeViewHolder(@NonNull View itemView) {
    super(itemView);
    mCoverView = findViewById(R.id.cover);
  }

  @Nullable
  @Override
  public OnImageShowListener getImageShowListener() {
    return mCoverView;
  }

  @Override
  public void bind(@NonNull WorksModel worksModel) {
    super.bind(worksModel);
    mCoverView.setAspectRatio(worksModel.mAspectRatio);
    mCoverView.getHierarchy().setPlaceholderImage(new ColorDrawable(worksModel.mColor));
  }
}

File: app/src/main/java/com/nb/core/test/subscribe/SubscribeViewHolder.kt
package com.nb.core.test.subscribe

import android.graphics.drawable.ColorDrawable
import android.view.View
import com.facebook.drawee.view.SimpleDraweeView
import com.hhh.onepiece.R
import com.nb.core.RecyclerViewHolder
import com.nb.core.test.model.WorksModel

class SubscribeViewHolder(view: View) : RecyclerViewHolder<WorksModel>(view) {

    private val coverView: SimpleDraweeView = itemView.findViewById(R.id.cover)

    override fun bind(model: WorksModel) {
        super.bind(model)
        coverView.aspectRatio = model.aspectRatio
        coverView.hierarchy.setPlaceholderImage(ColorDrawable(model.color))
    }
}